econplayground/main/models.py

```diff
@@ -229,7 +230,11 @@ class Meta(OrderedModel.Meta):
                                                       default='')
 
     x_axis_label = models.TextField(blank=True, null=True, default='')
+    x_axis_min = models.PositiveIntegerField(blank=True, null=True)
+    x_axis_max = models.PositiveIntegerField(blank=True, null=True)
     y_axis_label = models.TextField(blank=True, null=True, default='')
+    y_axis_min = models.PositiveIntegerField(blank=True, null=True)
+    y_axis_max = models.PositiveIntegerField(blank=True, null=True)
```

Comment:
These fields should default to 5 so all existing graphs will be using our new dynamic axis system. Also, we don't need x_axis_min or y_axis_min values, as it doesn't make sense to have a non-zero value here. In the jsxgraph config, this is in fact -0.02 to account for some graphical margin.
```
x_axis_max = models.PositiveIntegerField(default=5)
y_axis_max = models.PositiveIntegerField(default=5)
```

Reply:
`x_axis_min` and `y_axis_min` are there for instances where the graph might use a lower-bound greater than 0. Such a case might be where the instructor wants to give a better look at a small range of large numbers. They are already set to be non-zero as they are `PositiveIntegerField`s

Reply:
I mean non-zero as in a number other than zero - if the axis bar represents something like the number 3, the view is just quite confusing. There are many ramifications for a feature like this which is just not necessary to add right now.

Reply:
My mistake. I read that as non-negative